core/src/main/java/org/stellar/anchor/config/EventConfig.java

```diff
@@ -0,0 +1,13 @@
+package org.stellar.anchor.config;
+
+
+import java.util.Map;
+
+public interface EventConfig {
+  Map<String, String> getQueues();
+
+  String getKafkaBootstrapServer();
```

Comment:
These names are confusing to me...

getKafkaBootstrapServer returns a string instead of an error, so shouldn't it be something like `getKafkaBootstrapServerId()` or `getKafkaBootstrapServerName()`?

Also, getQueues() doesn't return a queue nor a queue map. What exactly is it about?

Reply:
getQueues() returns a map of event type to queue/topic name